libs/chiliui/chili/controls/object.lua

```diff
@@ -38,29 +38,37 @@ Object = {
 	children_hidden = {},
 	childrenByName = CreateWeakTable(),
 
-	OnDispose       = {},
-	OnClick         = {},
+	OnDispose       = {}, -- used frequently so not optimized out
+	OnClick         = {}, -- used frequently so also not optimized out
+
+	-- The following tables eat a lot of memory, and 99% of the time are empty, so are only left here as comments to know which ones can be used
+	-- Ensure to check that the table is not nil before inserting into them. 
```

Comment:
lgtm - did you check whole byar-chobby for usages of it ? Just see 1 edit in console.lua

Reply:
Yeah I checked whole of byarchobby for these.